infrastructure/helm/quantumserverless/README.md

```diff
@@ -59,6 +59,12 @@ Install from specific values file
 kubectl patch svc -n ray kuberay-apiserver-service --type json  --patch '[{"op" : "replace" ,"path" : "/spec/selector" ,"value" : {"app.kubernetes.io/component": "kuberay-apiserver"}}]'
 ```
 
+(temporary) Patch the kuberay-apiserver deployment
+
+```shell
+./hack/apisesrver=patch.sh <LOCAL-IP>
+```
```

Comment:
It's just one patch but it has long line.  I probably keep it.  The RP to remove this patch has been merged in the kuberay so this will be removed soon anyway.